tsl/test/sql/CMakeLists.txt

```diff
@@ -114,7 +114,7 @@ if((${PG_VERSION_MAJOR} GREATER_EQUAL "14"))
     list(APPEND TEST_FILES chunk_utils_internal.sql)
   endif()
   list(APPEND TEST_FILES compression.sql compression_update_delete.sql
-       compression_permissions.sql)
+       compression_permissions.sql recompress_chunk_segmentwise-v2.sql)
```

Comment:
really minor, just thought it might be useful to add an explanation why this is only tested for pg14 and later.

Reply:
Fixed, added comment.
There is no Custom Scan (HypertableModify) node for UPDATE/DELETE operations on PG versions < 14, and to support UPDATE/DELETE  operations on compressed chunks we need HypertableModify to be part of plan.
Thus we don't support this feature on PG12 and PG13.